dashboard/dashboard.scss

```diff
@@ -0,0 +1,21 @@
+body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins';
+    background-color: #F9FAFF; 
+}
+
+.main-wrapper {
+    background-color: #F9FAFF;     
```

Comment:
Would be better if u save all ur colors in variables and use it
Exapmle:
```
$main-bg: #f9faff;
$title-color: #202225;
$text-color: #a1abc9;
$input-color: #dce0ec;
$input-accent: #7297ff;
$input-error: #ff2567;
$white-color: #ffffff;
$online-color: #34c197;
$doctor-bg: #e3ebff;
```
and 
```
body {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
    font-family: 'Poppins';
    background-color: $main-bg; 
}
```